examples/qml_features/rust/src/properties.rs

```diff
@@ -21,20 +21,22 @@ pub mod qobject {
         // ANCHOR: book_properties_signature
         #[qobject]
         #[qml_element]
-        #[qproperty(bool, connected)]
-        #[qproperty(QUrl, connected_url)]
-        #[qproperty(QUrl, previous_connected_url)]
-        #[qproperty(QString, status_message)]
+        #[qproperty(bool, connected, READ, NOTIFY = url_signal)]
+        #[qproperty(QUrl, connected_url, READ, WRITE = set_url, NOTIFY = url_signal, RESET = reset_url)]
+        #[qproperty(QUrl, previous_connected_url, READ, NOTIFY = url_signal)]
+        #[qproperty(QString, status_message, READ, NOTIFY = url_signal)]
         type RustProperties = super::RustPropertiesRust;
         // ANCHOR_END: book_properties_signature
 
-        /// Connect to the given url
-        #[qinvokable]
-        fn connect(self: Pin<&mut RustProperties>, mut url: QUrl);
+        /// Custom on changed signal, used for all the properties
+        #[qsignal]
+        fn url_signal(self: Pin<&mut RustProperties>);
```

Comment:
hmm maybe `connected_state_changed` or something with changed in the name, then it would be `onConnectedStateChanged`